src/Commands/InitCommand.php

```diff
@@ -607,4 +596,41 @@ protected function publishWebLogin(): void
 
         file_put_contents(base_path('routes/web.php'), "\nAuth::routes();\n", FILE_APPEND);
     }
+
+    protected function publishAdminMigration(array $adminCredentials, string $adminPrefix, string $serviceName): array
+    {
+        if ($this->authType === AuthTypeEnum::Clerk) {
+            if (isset($this->adminCredentials['is_table_created'])) {
+                $viewName = 'admins_add_additional_admin';
+                $migrationName = "add_{$serviceName}_admin";
+
+                $adminCredentials['migrationName'] = Str::studly($migrationName);
+            } else {
+                $viewName = $migrationName = 'admins_create_table';
```

Comment:
not sure that such view is exists. Anyway, let's pusblish migration for the admins table creation outside of this method just after we decided that need to create user with the cleark auth